x-pack/plugins/observability_solution/synthetics/common/runtime_types/monitor_management/alert_config.ts

```diff
@@ -22,17 +21,6 @@ export const AlertConfigsCodec = t.partial({
   status: AlertConfigCodec,
 });
 
-export const AlertConfigSchema = schema.object({
-  tls: schema.maybe(
-    schema.object({
-      enabled: schema.boolean(),
-    })
-  ),
-  status: schema.object({
-    enabled: schema.boolean(),
-  }),
-});
```

Comment:
This file has some methods further below, that are used in some `.tsx` files... since runtime is required (not only a type import), the `@kbn/config-schema` will be injected...